index.js

```diff
@@ -368,6 +371,65 @@ module.exports = {
         // prettier-ignore
         if (group.excluded) return deferredSource.abort( new Error("We're excluded from this group, can't list members"))
 
+        if (allAdded) {
+          // TODO: make secrets live. box2 live get?
+          const secrets = group.readKeys.map((readKey) =>
+            readKey.key.toString('base64')
+          )
+
+          pull(
+            ssb.db.query(
+              where(
+                and(
+                  isDecrypted('box2'),
+                  type('group/add-member'),
+                  groupRecp(groupId)
+                )
+              ),
+              dbLive({ old: true }),
```

Comment:
this needed to be live for some reason :thinking:  but if we're in the group and have the group id then this msg should've already been decrypted, it's where we found out about the group to begin with

Reply:
simplified away this